packages/contracts/deploy/30_plugins/10_plugin-repos/00_create-plugins-repo.ts

```diff
@@ -22,17 +22,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // TokenVotingSetup
   await createPluginRepo(
     hre,
-    'TokenVoting',
+    'token-voting',
     'TokenVotingSetup',
     [1, 0, 0],
     '0x'
   );
 
   // AdminSetup
-  await createPluginRepo(hre, 'Admin', 'AdminSetup', [1, 0, 0], '0x');
+  await createPluginRepo(hre, 'admin', 'AdminSetup', [1, 0, 0], '0x');
```

Comment:
(These params will probably clash as soon as the PSP PR is merged)

Reply:
and? we will get conflicts either way